### PR TITLE
always use validators from a node

### PR DESCRIPTION
**Pull request by EduardoRFS**
## Problem

Currently the validator list on the sidecli are loaded from a hardcoded path, this means that if the files are in a different folder everything stops working.

## Solution

Always receives a node folder as input and read the `validators.json` from this folder, in practice this means that now `create-transaction` also has an additional folder_node  argument.

### CODE REVIEW

File: bin/sidecli.re [hunk 1]
```diff
@@ -13,17 +13,11 @@ let man = [
 
 // Todo from Andre: we may have to do peer discovery?
```
Comment (callistonianembrace) [inline bin/sidecli.re:14]: This TODO can be removed now I guess?

Comment (EduardoRFS) [reply bin/sidecli.re:14]: Nope, we still don't have peer discovery

**Review by callistonianembrace — APPROVED** [review]

File: bin/sidecli.re (hunk 1 above)
Comment (callistonianembrace) [reply bin/sidecli.re:14]: Validators will be sourced from the contract - not much to with p2p at the moment is it?

**Comment by callistonianembrace** [comment]
Should the setup.sh be updated @EduardoRFS ?

File: bin/sidecli.re (hunk 1 above)
Comment (callistonianembrace) [reply bin/sidecli.re:14]: (resolving as it is non blocking)

Comment (EduardoRFS) [reply bin/sidecli.re:14]: yes and validators shouldn't be required on the sidecli anyway

**Pull request merged**